platform-dependent-components/problem-solver/cxx/sections-module/agent/get_subdivision_agent.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * This source file is part of an OSTIS project. For the latest info, see http://ostis.net
+ * Distributed under the MIT License
+ * (See accompanying file COPYING.MIT or copy at http://opensource.org/licenses/MIT)
+ */
+
+#pragma once
+
+#include "sc-memory/kpm/sc_agent.hpp"
+
+#include <nlohmann/json.hpp>
```

Comment:
do we really need it here?

Reply:
without it the problem solver doesn't build

Reply:
eh?

i can't see usages of this lib here other than "using json = nlohmann::json;"

can you show me that error here or in PM pls?